.github/labeler.yml

```diff
@@ -0,0 +1,140 @@
+Docs:
+  - docs/*
+  - docs/_static/*
+  - docs/_templates/*
+  - docs/development/**/*
+  - docs/whatsnew/*
+  - examples/**/*
+  - licenses/*
+  - CITATION
+  - .mailmap
+  - readthedocs.yml
+  - '*.md'
+  - any: ['*.rst', '!CHANGES.rst']
+
+testing:
+  - astropy/tests/**/*
+  - codecov.yml
+  - conftest.py
+  - '**/conftest.py'
```

Comment:
Why the single-quotes on this one?

Reply:
Ditto on a few other examples below.  It should be harmless but it's not clear to me that they're needed either.

Reply:
It throws a YAML exception without the quotes.

Reply:
Oh, of course, because of the `*`!  Thank you.